server/db_mysql.go

```diff
@@ -195,6 +196,39 @@ func (ms *MsqlCache) GetFixity(start string, end string, item string, status str
 	return results
 }
 
+func (ms *MsqlCache) PostFixity(item string) (int, error)  {
+        var err error
+        fixity_results := ms.GetFixity("*", "*", item, "scheduled")
+        if len(fixity_results) > 0 {
+                err = ms.DeleteFixity(fixity_results[0].Id)
+                if err != nil {
+                        return 500, err
```

Comment:
Why is this returning integer error codes. These almost look like HTTP status codes....